src/mumble/Log_macx.mm

```diff
@@ -90,4 +93,5 @@ static bool growl_available() {
 		if (growl_available())
 			qt_mac_execute_apple_script(qsScript, NULL);
 	}
+#endif
 }
```

Comment:
The problem was the `else` in the code path above it.
I fixed the problem by moving the `#if` into the block.